packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -518,12 +520,12 @@ For a detailed explanation on deleting users, see [Delete (Deprovision)](/docs/c
 
 **POST** /Groups
 
-To create a Group object on the SCIM server, you need to push the Okta group using the Okta Admin Console:
+To create a Group object on the SCIM server, you first need to enable provisioning with the Group Push feature in the Admin Console:
 
 1. Select your SCIM integration from the list of integrations in your Okta org.
-1. On the **Push Groups** tab, click **Push Groups**.
+2. On the **Push Groups** tab, click **Push Groups**.
 
-You can select which existing Okta group to push, either by specifying a name or a rule. For more information, see [About Group push](https://help.okta.com/okta_help.htm?id=ext_Directory_Using_Group_Push) in the Okta Help Center.
+You can select which existing Okta group to push, either by specifying a name or a rule. If a group doesn't exist, create a new group in Okta and then push it to the SCIM server. For more information, see [About Group push](https://help.okta.com/okta_help.htm?id=ext_Directory_Using_Group_Push) in the Okta Help Center.
```

Comment:
Just a nit: I'm sure I'm missing something and I know I always have a problem of assuming that non-obvious things are obvious, so I'm operating on the assumption that I'm wrong here... but..

If we say that you can push okta groups to your scim app, does it really need to be made explicit that not only can you push the groups that were there when you logged in this morning, but also a new group that you create after lunch? especially in a dev doc

Reply:
@connorrowe-okta - I'm not sure I follow what change you're suggesting here. I'm basing this off the email you sent me where you said:
> Generally groups don't work quite the same way that users do. Group push requires provisioning to already be set up with the application - at that time (and during subsequent imports) group info for the app is brought into Okta. **When an admin goes to push an Okta group to the app they will see what groups already exist in the app and can either link to one of those or create a new group.** [emphasis mine] If a new group with the same name has been created in the SCIM app since the last import, it would cause a group push failure that could be retried after an import.

Reply:
@IanHakes-okta I don't know if I'm tracking how these are related - maybe you could explain what scenario you want to clarify here. Are we having people coming to us confused about how to create and push a new group to a SCIM app? To me that seems like the most straightforward/obvious case so I'm wondering if we're trying to highlight something else

Reply:
I haven't heard back from Andrea, so I'm going to mark this conversation as resolved for now and merge the changes we've gone over. If the updated docs don't meet her original request, I'll ask her to open a new doc ticket.